### ISSUE TITLE
Alignment of figure tops on GNU/Linux platform

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#31]
This issue was identified on the [Google Maps web app where the price is displayed in the left upper hand side of the page](https://www.google.com/maps/place/Boston+Marriott+Cambridge/@42.3630111,-71.0884194,17z/data=!4m10!3m9!1s0x89e370af6823c2ed:0x9179247fafa0ddf8!5m4!1s2019-07-31!2i4!4m1!1i2!8m2!3d42.3630072!4d-71.0862307):

<img width="203" alt="8afvf-image" src="https://user-images.githubusercontent.com/4249591/62233641-f5b11400-b396-11e9-9194-cd998f96532e.png">

(problem not shown in this image, this is from macOS and the problem seems to be Linux platform-specific?)

When viewed on the Linux platform by the reporter, the numerals do not align vertically for some sequential numeral price values.

I tested this with the unhinted woff2 file that is served by the Google Fonts API (production Google Sans) and our most recent v2.003 Google Sans Hinted ttf build using a local HTML/CSS site and can verify the following:

- this problem **does occur** on Ubuntu (tested on a macOS VM) when viewed in current versions of Chrome, Chromium, and Firefox browsers.
- this problem **does not occur** on macOS 10.14.4 when tested across current versions of Firefox and Chrome browsers
- this problem **does not occur** on WIn 10 when tested across current versions of Edge, Chrome, and Firefox browsers.

Here is an example from the Linux VM (generated by me in my testing) that shows what they are seeing:

### Unhinted woff2 build from Google Fonts API (currently serving to the Maps web app)

![VirtualBox_Ubuntu Linux_Unhinted](https://user-images.githubusercontent.com/4249591/62234243-4c6b1d80-b398-11e9-90a3-990f5d5aec4c.png)

### Hinted v2.003 ttf build from this repository

![VirtualBox_Ubuntu Linux_Hintedpng](https://user-images.githubusercontent.com/4249591/62234274-5d1b9380-b398-11e9-8f43-aa72ecad057a.png)

This looks like a hinting type problem and I can verify that the woff2 files are not hinted.  It occurs when I test with the hinted ttf builds as well.

Here is how they have the CSS implemented to lay out the numerals on that price line:

```css
.gm2-headline-6 {
    font-family: 'Google Sans';
    font-size: 1.125rem;
    font-weight: 400;
    line-height: 1.5rem;
}
```

They are setting the text at 1.125 rem units but I can verify that it occurs to varying degrees from 1.0 rem to 2.0 rem at 0.125 rem intervals.

Here is an archive with my test files that includes HTML/CSS site that replicates the above CSS with a couple of strings of numerals and the fonts that I used to test this:

[maps-tests.zip](https://github.com/Colophon-Foundry/google-sans/files/3453173/maps-tests.zip)

Thoughts?

**Comment by chrissimpkins** [comment]
See [original thread](https://github.com/Colophon-Foundry/google-sans/issues/95) for details.  This is [my summary](https://github.com/Colophon-Foundry/google-sans/issues/95#issuecomment-592777182) of what we found:

This issue ended up being a problem that was **caused by the FreeType autohinter on GNU/Linux platform only**.  We have numerals in the production max opsz designs that are slightly below the cap height level.  The curved numerals with overshoots snap to cap height, the numerals with straight top strokes snap to a pixel below cap height.  This leads to the staggered appearance across the numerals.  In GNU/Linux envs that default to slight hinting, it is not possible to implement manual instruction sets and address this problem.  So we are stuck from a hinting standpoint.  There is no hinting fix for that problem.

Marc Foley rec'd that we consider nudging the numerals a bit higher to move past the (unknown to me) boundary that indicates the shape should snap to the cap height level.

**Comment by chrissimpkins** [comment]
Related #344

**Comment by chrissimpkins** [comment]
> this problem does not occur on WIn 10 when tested across current versions of Edge, Chrome, and Firefox browsers.

I reviewed our v6.000 builds in a DirectWrite environment and can re-confirm that this issue does not occur in recent Win env's with our latest files.

**Comment by MariannaPaszkowska** [comment]
Will be resolved by #412

**Comment by belluzj** [comment]
Closed by https://github.com/googlefonts/googlesans/pull/412
